packages/devtools-license-check/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "devtools-license-check",
+  "version": "0.1.0",
+  "description": "Verify licenses for inclusion in Mozilla DevTools",
+  "repository": "github:devtools-html/devtools-core",
+  "author": "",
```

Comment:
Missed that in the other review, but maybe fill in the author field with your name?